qa/SidechainTestFramework/account/smart_contract_resources/contracts/ForgerStakesV2.sol

```diff
@@ -0,0 +1,116 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/*
+ Native Contract managing forgers stakes - Version 2 (activated from EON version 1.4)
+ contract address: 0x0000000000000000000022222222222222222333
+*/
+interface ForgerStakesV2 {
+
+    // Event declaration
+    // Up to 3 parameters can be indexed.
+    // Indexed parameters helps you filter the logs by the indexed parameter
+    event RegisterForger(address indexed sender,  bytes32 signPubKey, bytes32 indexed vrf1, bytes1 indexed vrf2, uint256 value, uint32 rewardShare, address reward_address);
+    event UpdateForger(address indexed sender, bytes32 signPubKey,  bytes32 indexed vrf1, bytes1 indexed vrf2,  uint32 rewardShare, address reward_address);
+    event DelegateForgerStake(address indexed sender, bytes32 signPubKey, bytes32 indexed vrf1, bytes1 indexed vrf2, uint256 value);
+    event WithdrawForgerStake(address indexed owner, bytes32 signPubKey, bytes32 indexed vrf1, bytes1 indexed vrf2, uint256 value);
```

Comment:
Maybe, to be consistent with the others events, `owner` can be renamed `sender`.